.storybook/webpack.config.js

```diff
@@ -1,8 +1,14 @@
 let babel = require('./babel.config.js');
 let config = require('./../.config/webpack/base.js')({
-    babel
+    babel,
+    preprocessor: {
+        variables: {
+            mode: 'eager'
+        }
+    }
 });
 
 config.externals = {};
+delete config.plugins;
```

Comment:
We don't want that weird path resolution code - this runs a server-side React generation